test/index.ts

```diff
@@ -13,4 +13,5 @@ after(async () => {
   await server.stop();
 });
 
-import './login-mutation-test';
+//import './login-mutation-test';
```

Comment:
normalmente a gente evita deixar codigo comentado, se for usar depois daria pra recuperar pelo historico do git, então em teoria da pra remover